application/models/Question.php

```diff
@@ -295,8 +295,10 @@ public function getAdvancedSettingsWithValues($iQuestionID, $sQuestionType, $iSu
         // TODO: move getQuestionAttributesSettings() to QuestionAttribute model to avoid code duplication
         $aAttributeNames = QuestionAttribute::getQuestionAttributesSettings($sQuestionType);
 
-        // If the question has a custom template, we first check if it provides custom attributes
+        // for questioneditor and public question view. Used QuestionBaseDataSet and Survey_Common_Action
+        //$aAttributeNames = array_merge($aAttributeNames, QuestionAttribute::getQuestionAttributesPlugins($sQuestionType));
```

Comment:
In my test : this work without this one … need more testing, maybe related to question type